PassPy/password_manager.py

```diff
@@ -81,9 +78,14 @@ def delete_pass():
     if login == None:
         print('\nThat login doesn''t exist')
     else:
-        del data[data_from]
-        with open(passwords_path, 'w') as file_data:
-            json.dump(data, file_data)
+        accept = input(
+            'Are you sure you wish to delete the password from:{}Y/n'.format(data_from)).upper()
```

Comment:
F-strings are more readable that `.format`
https://stackoverflow.com/questions/43123408/f-strings-vs-str-format/43124267

```suggestion
        accept = input(f'Are you sure you wish to delete the password from:{data_from} Y/n').upper()
```